src/engine/strat_engine/backstore/util.rs

```diff
@@ -82,7 +89,7 @@ pub fn get_stratis_block_devices() -> StratisResult<Vec<PathBuf>> {
 
         Ok(get_all_empty_devices()?
             .into_iter()
-            .filter(|x| is_stratis_device(&x).ok().is_some())
+            .filter(|x| is_stratis_device(&x).unwrap_or(None).is_some())
```

Comment:
AFAICT, this is just a patch to make ```get_stratis_block_devices()``` match its specification a bit better and doesn't really have anything to do with multipath. Please remove it from this commit.